platform-http-proxy/src/main/java/org/codice/ddf/platform/http/proxy/HttpProxy.java

```diff
@@ -81,25 +90,29 @@ public void startProxy() throws Exception {
                 host = "localhost";
             }
             endpointName = ((HttpProxyServiceImpl) httpProxyService)
-                    .start("0.0.0.0:" + httpPort, "https://" + host + ":" + httpsPort, 1000, true);
+                    .start("0.0.0.0:" + httpPort, "https://" + host + ":" + httpsPort, 120000, true,
```

Comment:
Yeah that is my bad. I changed it to fix a different issue I was seeing in the integration tests and did a quick build :) I'll update the unit tests.